src/app/signup/page.jsx

```diff
@@ -0,0 +1,97 @@
+import Image from "next/image";
+import styles from "../stylesheet/signup.module.css";
+import SocialLoging from "../components/SocialLogin";
+import TextInput from "../components/TextInput";
+import Label from "../components/Label";
+
+export default function signUp() {
+  return (
+    <main className={styles.main}>
+      <div className={styles.container}>
+        <Image
+          className={styles.logo}
+          src="/Frame.svg"
+          alt="Ember logo"
+          width={135}
+          height={54}
+          priority
+        />
+        <h2>Sign up for an account.</h2>
+        <p className={styles.subtitle}>Send, spend and save smarter</p>
+        <SocialLoging />
+        <div className={styles.login_email}>
+          <div className={styles.login_email__line}>
+            <hr />
+          </div>
```

Comment:
You can style the hr and save lines 😄 
```suggestion
          <hr className={styles.login_email__line} />
```